index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Pele-the magician</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+      <div class="contents">
+      <h1 id="title">Pele,Edson Arantes do Nascimento</h1>
+      <h2 id="subtitle">The magician who made people love football</h2>
+      <div id="img-div">
+        <image id="image" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTIngNS3t27r2Qe8uXoYmjP8tFJHFLf93FLlw&usqp=CAU" />
+        <figcaption id="img-caption"> Pele(1940-2022)</figcaption>
+      </div >
+
+      <div id="tribute-info">
+	
+       <h2>Pele Fast Facts</h2>  
+
+       <ul>
+
+        <li>
+          Pele's real name is Edson Arantes do Nascimento, but he is widely known by his nickname Pele.
+        </li>
+
+        <li>
+          He began his professional soccer career as a teenager with Santos FC in Brazil and quickly established himself as one of the best players in the world.
+        </li>
+
+        <li>
+          Pele helped Santos win multiple Brazilian league titles and international club competitions.
+        </li>
+
+        <li>
+          He was the first player to score over 1,000 goals in his career and set a number of records that still stand today
+        </li>
+
+        <li>
+          Pele was also known as "The King of Football" and "The Black Pearl"
+        </li>
+       </ul>
+        
+        </div>
+
+        <div>
+          <h2>The records of Pele:</h2>
+          <ul>
+            <li>A World Cup legend</li>
+            <li>Most goals</li>
+            <li>Most goals in a calendar year</li>
+            <li>Most hat-tricks</li>
+            <li>Youngest player to appear in, score in, and win a FIFA World Cup final</li>
+            <li>Brazil’s joint top goal scorer</li>
+          </ul>
+        </div>
+
+        <div id="tribute-info">
+          <p>For more information you can check his <a id="tribute-link" target="_blank" href="https://en.wikipedia.org/wiki/Pel%C3%A9">Wikipedia Page</a></p>
+          </div>
+        </div>
+
+        
+
+
+      
+    </main>
+    
+    <script  src="js/index.js"></script>
```

Comment:
Please remove this line as you do not have any javascript file in your repository.